src/pam-ssh-add/Makefile.am

```diff
@@ -1,17 +1,19 @@
 noinst_LIBRARIES += libpam_ssh_add.a
 
 libpam_ssh_add_a_SOURCES = \
+	src/common/cockpitcloserange.h \
+	src/common/cockpitcloserange.c \
 	src/pam-ssh-add/pam-ssh-add.c \
 	src/pam-ssh-add/pam-ssh-add.h \
 	$(NULL)
 
-libpam_ssh_add_a_CFLAGS = $(NULL)
+libpam_ssh_add_a_CFLAGS = -fPIC
 
 libpam_ssh_add_a_LIBS = \
 	libpam_ssh_add.a \
 	$(PAM_LIBS)
 
-pam_ssh_add_FILES = src/pam-ssh-add/pam-ssh-add.c src/pam-ssh-add/pam-ssh-add.c
+pam_ssh_add_FILES = src/pam-ssh-add/pam-ssh-add.c
```

Comment:
Well, that's nice I guess?

Reply:
Right, this is unrelated cleanup. Split out into a separate commit.